Gemfile

```diff
@@ -6,6 +6,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 gemspec
 
+gem 'activesupport', '>= 4.0'
```

Comment:
This is needed for `classify`, to convert from model file name to class name. I suspect most folks who have this package installed already have Rails, so this shouldn't be a big deal?

Reply:
This would need to be a runtime dependency then (in the gemspec), wouldn't it?

Reply:
Good call! Done in d776f5c